archeryutils/rounds.py

```diff
@@ -211,6 +214,12 @@ def __init__(  # noqa: PLR0913
     ) -> None:
         self.name = name
         self.passes = list(passes)
+        if all(False for _ in self.passes):
```

Comment:
We check that the contents are actual passes later so this might be more readable as:
```python
if len(self.passes) == 0:
   ...
```
Which is more explict about what we are checking for, but then that begs the question of why not just do the traditional python approach:
```python
if not self.passes:
```

Reply:
Huh, could've sworn I tried that and it had a problem but seems to work now 🤷 Amended and improved test in https://github.com/jatkinson1000/archeryutils/pull/84/commits/8f3dacfc1c66dce2764359a23a8b8c2b617edcae.